resharper/src/resharper-unity/Rider/UnityOptionsPage.cs

```diff
@@ -27,6 +27,7 @@ public class UnityOptionsPage : OptionsPageBase
             Header("General");
 
             CheckBox((UnitySettings s) => s.InstallUnity3DRiderPlugin, "Install or update Rider plugin automatically");
+            CheckBox((UnitySettings s) => s.AllowAutomaticRefreshInUnity, "Allow Rider to do \"Refresh in Unity\" automatically");
```

Comment:
"Automatically refresh Unity"?

Reply:
Automatically refresh Assets in Unity?

Reply:
I think "Automatically refresh assets in Unity" is close enough to auto refresh that Unity people will understand what it does, but includes a bit more context about what gets refreshed.